django_prbac/models.py

```diff
@@ -65,28 +74,101 @@ class Role(ValidatingModel, models.Model):
     # Methods
     # -------
 
+    @classmethod
+    def get_cache(cls):
+        try:
+            cache = cls.cache
+        except AttributeError:
+            timeout = getattr(settings, 'DJANGO_PRBAC_CACHE_TIMEOUT', 60)
```

Comment:
The default cache timeout is 1 minute. This is the amount of time that a role or grant change will take to become effective. We could get fancier and make the model managers clear the cache on write operations if the cache duration becomes a problem.